server/cmd/mmctl/commands/permissions.go

```diff
@@ -42,7 +42,7 @@ var RemovePermissionsCmd = &cobra.Command{
 
 var ShowRoleCmd = &cobra.Command{
 	Use:        "show <role_name>",
-	Deprecated: "please use \"role show\" instead",
+	Deprecated: "please use \"mmctl permissions role show\" instead",
```

Comment:
I found the deprecation warning a bit hard to read. 

**New**
```bash
$ mmctl permissions show system_user

Command "show" is deprecated, please use "mmctl permissions role show" instead
...
```

**Old**
```bash
$ mmctl permissions show system_user

Command "show" is deprecated, please use "role show" instead
...
```

Reply:
Please let me know what you think. If you like the change, I will applied it to the other deprecated commands.

Reply:
@mgdelacroix What do you think?

Reply:
@hanzei when I saw it at first I was about to disagree, as I think we should provide instructions as simple as possible, but then I saw your examples with the command just atop, and I think it's easy enough to understand what should be changed and how.

Honestly I still think the first approach causes a bit less confusion than the second (most of the time when you're changing the command and don't want to think too much), but nothing that can't be solved going back, reading with a bit more focus and trying again. I'm just 1/5, so I'm perfectly happy with the change if you feel strongly about it.

Reply:
Should we apply the same change to the examples? Take, e.g. the `mmctl channel rename` command:

![Screenshot from 2024-10-17 13-38-37](https://github.com/user-attachments/assets/91d409f8-f494-4724-b6fe-c1a85b4f54c3)

The Usage does include `mmctl` as a prefix, but the examples don't. The help text could be clearer by prefixing the root command.

Reply:
0/5 if we want to do that as part of this PR. This conversation is extending beyond the original scope of the PR.